app/src/main/java/org/fossasia/openevent/app/data/UtilModel.java

```diff
@@ -43,6 +46,25 @@ public void saveString(String key, String value) {
     }
 
     @Override
+    public Set<String> getStringSet(String key, Set<String> defaultValue) {
+        return sharedPreferences.getStringSet(key, defaultValue);
+    }
+
+    @Override
+    public void saveStringSet(String key, Set<String> value) {
+        SharedPreferences.Editor editor = sharedPreferences.edit();
+        editor.putStringSet(key, value);
+        editor.apply();
+    }
+
+    @Override
+    public void addStringSetElement(String key, String value) {
+        Set<String> set = sharedPreferences.getStringSet(key, new HashSet<>());
```

Comment:
A minor change required here, use the `getStringSet` of UtilModel, so that there is less dependence on external dependencies